db.js

```diff
@@ -0,0 +1,66 @@
+import mongoose from 'mongoose'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+mongoose.set('strictQuery', true)
+
+async function dbClose() {
+    await mongoose.connection.close()
+    console.log("Database disconnected!")
+}
+
+// Connect to MongoDB via Mongoose
+try {
+    const m = await mongoose.connect(process.env.ATLAS_DB_URL)
+    console.log(m.connection.readyState === 1 ? 'Mongoose connected!' : 'Mongoose failed to connect')
+}
+catch (err) {
+    console.log(err)
+}
+
+// Define Book schema
+const bookSchema = new mongoose.Schema({
+    title: { type: String, required: true}, 
+    author: { type: String, required: true}
+})
+
+// Create book model based on schema
+const BookModel = mongoose.model('Book', bookSchema)
+
+// Define Appointment schema
+const appointmentSchema = new mongoose.Schema({
```

Comment:
divide name to first name and last name?
inc_book and out-book use objectID from the book schema?
add time?
remove location as it contains in out-book?

Reply:
amended